x/participationrewards/keeper/keeper_test.go

```diff
@@ -123,7 +123,12 @@ func (s *KeeperTestSuite) coreTest() {
 		return false
 	})
 
-	_, found := quicksilver.ClaimsManagerKeeper.GetLastEpochClaim(ctx, "cosmoshub-4", "quick16pxh2v4hr28h2gkntgfk8qgh47pfmjfhzgeure", cmtypes.ClaimTypeLiquidToken, "osmosis-1")
+	// In the setupTestClaims() the claim was set in the store by ClaimsManagerKeeper.SetClaim
+	// which use the GetKeyClaim key to set in the store instead of GetKeyLastEpochClaim so
+	// it will return not found when use the GetLastEpochClaim
+	// either we use the ClaimsManagerKeeper.SetLastEpochClaim to add the last claim or
+	// switch to GetClaim in the line of code below
+	_, found := quicksilver.ClaimsManagerKeeper.GetClaim(ctx, "cosmoshub-4", "quick16pxh2v4hr28h2gkntgfk8qgh47pfmjfhzgeure", cmtypes.ClaimTypeLiquidToken, "osmosis-1")
```

Comment:
An epoch has passed here, (line 116) so this should have transitioned to GetLastEpochClaim; unless the epoch is panicking and being caught by the panic handler (which is a pain in testing because we probably want panicking epoch to case tests to fail!) @ajansari95

Reply:
makes sense to panic

Reply:
i think we know why. Its because the ibc testchainID add another surfix in v7 so testchain1 should be testchain1-1 in the `setupTestProtocolData` func

Reply:
reverted back to normal